mobly/test_runner.py

```diff
@@ -472,7 +421,7 @@ def run_test_class(self, test_cls_name, test_cases=None):
                 self.results += e.results
                 raise e
 
-    def run(self, test_class=None):
+    def run(self, test_classes):
```

Comment:
It was used extensively in Android. It's the basis for executing multiple test classes in one command (-tc XxxTest YyyTest)
The intention is that users can easily construct their test suite in code, similar to in unittest. More work is needed to document this, but leaving the hook here is useful